IdentityCore/src/requests/sdk/MSIDTokenResponseValidator.m

```diff
@@ -172,7 +172,9 @@ - (MSIDTokenResult *)validateAndSaveBrokerResponse:(MSIDBrokerResponse *)brokerR
     MSIDConfiguration *configuration = [[MSIDConfiguration alloc] initWithAuthority:brokerResponse.msidAuthority
                                                                         redirectUri:nil
                                                                            clientId:brokerResponse.clientId
-                                                                             target:brokerResponse.target];
+                                                                             target:brokerResponse.target
+                                                           nestedAuthBrokerClientId:brokerResponse.nestedAuthBrokerClientId
+                                                        nestedAuthBrokerRedirectUri:nil];
```

Comment:
Why redirectURI is nil here?

Reply:
I was following the same pattern in that same call, the `redirectUri` parameter is also nil.